SQL Assignments/Assign4.sql

```diff
@@ -0,0 +1,359 @@
+-- SOLUTIONS-
+
+-- Write a query to get unique department ID from employee table.
+select distinct DEPARTMENT_ID from employees;
+
+-- Write a query to get all employee details from the employee table order by first name, descending
+select * from employees order by FIRST_NAME;
+
+-- Write a query to get the employee ID, names (first_name, last_name), salary in ascending order of
+-- salary.
+select EMPLOYEE_ID, FIRST_NAME, LAST_NAME, SALARY from employees order by SALARY;
```

Comment:
can you try updating the query to get names as a single column ?

Reply:
I can, But In question its said to give all the details.

select FIRST_NAME, LAST_NAME from employees order by SALARY;
Do you mean like this?

Reply:
In the question it is mentioned names as (first_name, last_name). Can you try that ?

Reply:
Ohk done, I have commited the changes and pushed the repo.